### DISCUSSION TITLE
Difference between multiple queues and using priority on a single queue

### DISCUSSION THREAD

**Discussion by leetice** [Mergifyio/mergify#5043] [Q&A]
Curious, is there any functional difference in defining two queues (e.g. `urgent` & `default`) and sorting PRs onto them accordingly, as opposed to having a single queue (`default`) and assigning different priorities for PRs accordingly (e.g. `high` & `low`)?  

Thanks!

**Comment by jd** [comment]
Hi @leetice!

The main difference is that jobs running into your `default` queue will get interrupted when a PR enters the `urgent` queue.
This is not true when using a single queue with priority inside the queue: the ongoing PR being tested will not be interrupted.

(Using multiple queues also allows you to have differents settings (spec checks, batches, etc))

**Reply by leetice** [reply]
Thank you, Julien! That's extremely helpful, and thanks also for the quick response!
